packages/core/src/Actor.ts

```diff
@@ -83,11 +83,11 @@ export function createDeferredActor(
 
   if (isMachine(entity)) {
     // "mute" the existing service scope so potential spawned actors within the `.initialState` stay deferred here
-    // @ts-ignore
-    tempActor.state = serviceScope.provide(
```

Comment:
we might need both properties here for the time being - removing `.state` is a compatibility risk with `@xstate/react` etc

Reply:
Done in b256bf3bc1a9ff02f8c81d594f5c9bc691b6e3e5